core/src/qos_service.rs

```diff
@@ -128,6 +143,16 @@ impl QosService {
                 Ok(current_block_cost) => {
                     debug!("slot {:?}, transaction {:?}, cost {:?}, fit into current block, current block cost {}", bank.slot(), tx, cost, current_block_cost);
                     self.metrics.selected_txs_count.fetch_add(1, Ordering::Relaxed);
+                    self.report_sender
+                        .send(QosMetrics::TransactionCost {
+                            slot: bank.slot(),
+                            tx_signature: *tx.signature(),
+                            signature_cost: cost.signature_cost,
+                            write_lock_cost: cost.write_lock_cost,
+                            data_bytes_cost: cost.data_bytes_cost,
+                            execution_cost: cost.execution_cost,
+                        } )
+                        .unwrap_or_else(|err| warn!("qos service report tx cost details failed: {:?}", err));
```

Comment:
not super comfortable of sending every transaction cost details to influx, so just sending the needed CUs for now.